config/locales/controllers/groups/en.yml

```diff
@@ -0,0 +1,5 @@
+en:
+  groups:
+    launch:
+      notice_invite_mentor: "Group mentor can not be invited."
```

Comment:
Can we use this terminology as we can have multiple mentors per group?

Reply:
This PR is post the multiple mentor feature release, will make the required changes!